### DISCUSSION TITLE
Connecting New Database

### DISCUSSION THREAD

**Discussion by MLemke24** [hasura/graphql-engine#9277] [Question]
I am following the quick start guide to set up a Hasura instance with Docker. I took the PG_DATABASE_URL they recommend since I dont have a database setup but I get this error everytime: 

[
    {
        "definition": "golfdatatracker",
        "message": "could not translate host name \"postgres\" to address: Temporary failure in name resolution\n",
        "name": "source golfdatatracker",
        "reason": "Inconsistent object: connection error",
        "type": "source"
    }
]

The URL I am using: PG_DATABASE_URL:  postgres://postgres:postgrespassword@postgres:5432/postgres which is the recommendation.

Am I doing something wrong?

**Comment by adas98012** [comment]
@MLemke24 
After running the docker, do you see a container running named "postgres" ?

**Comment by MLemke24** [comment]
Yeah the docker image seems to be running fine:
[image: Screen Shot 2022-12-08 at 4.03.37 PM.png]

However I just get this error no matter what i do:
[image: Screen Shot 2022-12-08 at 4.03.26 PM.png]


Could it possibly be due to the fact that I am running this on an M1
Macbook?

**Comment by adas98012** [comment]
@MLemke24 
Somehow, I am unable to see the screenshots that you attached to your last post.
What is the version of the Hasura GraphQL engine you are using ?
Also, please share your docker-compose.yaml file (redact any sensitive info there), so we can troubleshoot.
